{{cookiecutter.project_name}}/docker-compose.override.yml

```diff
@@ -11,6 +11,21 @@
 
 version: "3.8"
 services:
+
+  db:
+    image: "postgres:14-alpine"
+    restart: unless-stopped
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    networks:
+      - webnet
```

Comment:
It is better to create separate networks for each infra object: one for `redis` and celery, one for `db`.

This way you have more control over network security.